From ac6ba9a14be2caa8b3b4cb051138b77521ce4b35 Mon Sep 17 00:00:00 2001
From: David Hildenbrand <david@redhat.com>
Date: Tue, 17 Oct 2017 19:16:04 +0200
Subject: [PATCH 59/69] tools/kvm_stat: use variables instead of hard paths in
 help output

RH-Author: David Hildenbrand <david@redhat.com>
Message-id: <20171017191605.2378-39-david@redhat.com>
Patchwork-id: 77346
O-Subject: [RHEL-7.5 qemu-kvm-rhev PATCH 38/39] tools/kvm_stat: use variables instead of hard paths in help output
Bugzilla: 1497137
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>

Upstream-status: linux.git efcb521943a8df5210f16f312037c2edc3e1449f

commit efcb521943a8df5210f16f312037c2edc3e1449f
Author: Lin Ma <lma@suse.com>
Date:   Tue Jul 25 19:05:53 2017 +0800

    tools/kvm_stat: use variables instead of hard paths in help output

    Using variables instead of hard paths makes the requirements information
    more accurate.

    Signed-off-by: Lin Ma <lma@suse.com>
    Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>

Signed-off-by: David Hildenbrand <david@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 scripts/kvm/kvm_stat | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/kvm/kvm_stat b/scripts/kvm/kvm_stat
index dd8f00c..5704044 100755
--- a/scripts/kvm/kvm_stat
+++ b/scripts/kvm/kvm_stat
@@ -1413,8 +1413,8 @@ performance.
 
 Requirements:
 - Access to:
-    /sys/kernel/debug/kvm
-    /sys/kernel/debug/trace/events/*
+    %s
+    %s/events/*
     /proc/pid/task
 - /proc/sys/kernel/perf_event_paranoid < 1 if user has no
   CAP_SYS_ADMIN and perf events are used.
@@ -1434,7 +1434,7 @@ Interactive Commands:
    s     set update interval
    x     toggle reporting of stats for individual child trace events
 Press any other key to refresh statistics immediately.
-"""
+""" % (PATH_DEBUGFS_KVM, PATH_DEBUGFS_TRACING)
 
     class PlainHelpFormatter(optparse.IndentedHelpFormatter):
         def format_description(self, description):
-- 
1.8.3.1

